nodejs/week1/homework/src/backend/app.js

```diff
@@ -0,0 +1,58 @@
+const express = require("express");
+const app = express();
+
+// import data here
+const meals = require("./data/meals.json");
+const reservations = require("./data/reservations.json");
+const reviews = require("./data/reviews.json");
+
+// this is where you will be adding your routes
+app.get("/", async (request, response) => {
+  response.send("Meal Sharing Web App");
+});
+
+//meals
+
+const mealWithRev = meals.map((m) => {
+  const rev = reviews.filter((r) => r.id === m.id);
+  if (rev) {
+    m.reviews = rev;
+  } else {
+    m.reviews = " ";
```

Comment:
This statement doesn't seem necessary, in case you want to initialize reviews for all meals, [] is a better choice than " " (why?)

Reply:
I did it, because maybe  a new meal haven’t  got any reviews yet